config/default.js

```diff
@@ -118,7 +118,7 @@ module.exports = {
     notificationEmail: '',
   },
   scienceBeam: {
-    url: 'https://sciencebeam-texture.elifesciences.org/api/convert',
```

Comment:
it's good to take out this, but since tests depend on it you either need to stub it or to target a local container in `docker-compose.yml`?

Reply:
Captured in #1287